docs/settingup.md

```diff
@@ -700,7 +700,7 @@ Publish bookmark previously created in the script with an id `bookmark1`.
 }
 ```
 
-Publish bookmark with title "bookmark of testuser", where `testuser` is the username of the simulated user. 
+Publish the bookmark with the title "bookmark of testuser", where "testuser" is the username of the simulated user.
```

Comment:
Could you please "highlight" title (`title`), so that it corresponds to the way it is presented under Settings?